src/content/admin/cloud-credentials/aws.mdx

```diff
@@ -0,0 +1,123 @@
+---
+title: AWS
+description: Configure AWS credentials
+sidebar_label: aws
+id: aws-cloud-credentials
+---
+import Image from "@theme/Image";
+
+
+This tutorial will guide you on how to configure AWS credentials for your Okteto instance. In this specific guide we'll request access to s3 resource but you can provide access to any other AWS resource by using a role with more/different permissions.
+
+It's recommended that you have the [AWS CLI installed](https://docs.aws.amazon.com/cli/latest/userguide/getting-started-install.html#getting-started-install-instructions) to follow this tutorial.
+
+AWS credentials are configured using OIDC federation to assume an IAM Role via Web Identity.
```

Comment:
Can we add links to what "OIDC federation" and "IAM Role via Web Identity" means?